docs/reference/runtimes/python/python-reference.md

```diff
@@ -40,13 +40,34 @@ COPY --from=processor /home/nuclio/bin/processor /usr/local/bin/processor
 COPY --from=processor /home/nuclio/bin/py /opt/nuclio/
 COPY --from=uhttpc /home/nuclio/bin/uhttpc /usr/local/bin/uhttpc
 
-# Copy the handler directory to /opt/nuclio
-COPY . /opt/nuclio
+RUN pip install nuclio-sdk msgpack --no-index --find-links /opt/nuclio/whl
 
 # Readiness probe
 HEALTHCHECK --interval=1s --timeout=3s CMD /usr/local/bin/uhttpc --url http://127.0.0.1:8082/ready || exit 1
 
+# USER CONTENT
+ADD ./my-function-code /opt/nuclio
+ADD ./my-function.yaml /etc/nuclio/config/processor/processor.yaml
+# END OF USER CONTENT
+
 # Run processor with configuration and platform configuration
 CMD [ "processor" ]
 ```
 
+### Notes
+
+- Make sure to change both `my-function-code` & `my-function.yaml` to your real function code & configuration
+- `my-function.yaml` should at least include your handler & runtime, in example:
+```yaml
+spec:
+  handler: main:handler
+  runtime: python:3.6
+
+```
+
+Building & Running example commands:
```

Comment:
I would replace this part with a "Build and execution example" section at the end of the file (although this information is more of a "task" than a configuration "reference"):
```markdown
<a id="build-and-execution"></a>
## Build and execution

Following are example commands for building and running the latest version of a `my-function` function that's listening on port 8090; replace the function name and version and the port number, as needed:
```
&#96;&#96;&#96;sh
docker build -t my-function:latest .
docker run --name my-function --rm -d -p 8090:8080 my-function:latest
&#96;&#96;&#96;
"
This should produce output similar to the following:
"
## Build and execution

Following are example commands for building and running the latest version of a `my-function` function that's listening on port 8090; replace the function name and version and the port number, as needed:
```sh
docker build -t my-function:latest .
docker run --name my-function --rm -d -p 8090:8080 my-function:latest
```
"

In addition, add the following at the end of the "In this document" section:
```markdown
- [Build and execution](#build-and-execution)
```